README.md

```
@@ -37,12 +37,59 @@ This repository contains a sleek, reactive frontend, but it leans on a robust ba
+
+### Minimal config
+```yaml
+    type: custom:item-list-card
+    filter_items_entity: sensor.kellervorrate_combined_filtered_items
+    hash_entity: sensor.kellervorrate_combined_filtered_items_hash
+    shopping_list_entity: todo.einkaufsliste
+    filter_entity: input_text.search_todo_list
+```
+
+
+### Full config
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)